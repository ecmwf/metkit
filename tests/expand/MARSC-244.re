retrieve,
DOMAIN=G,DATE=20220501,TIME=0000,EXPVER=0001,CLASS=CE,TYPE=FC,STREAM=EFAS,ORIGIN=ECMF,LEVTYPE=SFC,STEP=6,MODEL=LISFLOOD,PARAM=260267,
    target="reference.RgyKKl.data"