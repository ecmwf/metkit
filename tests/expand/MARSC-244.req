retrieve,
    DATE=20130225,TIME=0000,ORIGIN=SRNWPPEPS-DWD-EUA,STEP=3,LEVTYPE=SFC,NUMBER=1,PARAM=166,EXPVER=test,CLASS=TI,MODEL=LAM,TYPE=PF,STREAM=ENFO,
    target=reference.data