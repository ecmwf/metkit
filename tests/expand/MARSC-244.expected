RETRIEVE,
    CLASS      = CE,
    TYPE       = FC,
    STREAM     = EFAS,
    EXPVER     = 0001,
    MODEL      = LISFLOOD,
    REPRES     = SH,
    STEP       = 3,
    NUMBER     = 1,
    ORIGIN     = SRNWPPEPS-DWD-EUA,
    TARGET     = reference.3YwXRI.data,
    PADDING    = 0,
    PROCESS    = LOCAL 